README.md

```diff
@@ -4,76 +4,110 @@
 
 ![logo](http://kuzzle.io/themes/kuzzleio/images/kuzzle-logo-blue-500.png)
 
-**A backend software, self-hostable and ready to use to power modern apps**
+## Why Kuzzle ?
 
-# Installation
+Kuzzle is a ready-to-use, **on-premises backend** that enables you to manage your persistent data and be notified in real-time on whatever happens to it. It also provides you with a flexible and powerful user-management system.
 
-## Using Docker
+Kuzzle enables you to build modern web applications and complex IoT networks in no time.
+
+* **Persisted data**: store your data and perform advanced searches on it.
+* **Real-time notifications**: subscribe to fine-grained subsets of data.
+* **User Management**: login, logout and security rules are no more a burden.
+* **Extensible**: fit Kuzzle to your needs by leveraging the plugin system.
+
+
+## Installation
+
+### Docker install
 
 If you are running Docker and just want to get your own Kuzzle running, you can use the provided docker-compose file.
 
 Prerequisites:
 
-* [Docker](https://docs.docker.com/engine/installation/) (version >1.10.0)
-* [Docker Compose](https://docs.docker.com/compose/install/) (version >1.8.0)
+* [Docker](https://docs.docker.com/engine/installation/) (version >1.13.0)
```

Comment:
this docker compose only [requires compose 1.7+ and docker engine 1.10+](https://github.com/docker/compose/releases/tag/1.7.0)

The given requirements are true for the dev compose stack, which relies on version 3 of compose (cf https://docs.docker.com/compose/compose-file/compose-versioning/)